<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Sanjeet3149591" name="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest" tests="11" failures="1" timestamp="17 Dec 2018 16:26:34 GMT" time="14.767" errors="0">
  <testcase name="placeNewOrderInPresent" time="0.508" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="cancelOrder" time="0.420" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="fetchTheOrderDetails" time="0.896" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="takeTheOrderDetails" time="0.286" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="CompleteTheOrderDetails" time="0.393" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="CompleteTheNewOrder" time="2.344" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="CompleteTheOrderAfterCompleted" time="1.941" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="cancelTheOrderAfterCompleted" time="4.301" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="cancelTheOrderAfterCancel" time="1.792" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest">
    <failure type="java.lang.AssertionError" message="Status code is not 422 expected [422] but found [200]">
      <![CDATA[java.lang.AssertionError: Status code is not 422 expected [422] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest.CommonUtilityForResponseCode422(TestcasesForMoreThan3StopsInPresentFutureTest.java:250)
at com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest.cancelTheOrderAfterCancel(TestcasesForMoreThan3StopsInPresentFutureTest.java:317)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- cancelTheOrderAfterCancel -->
  <testcase name="CompleteTheOrderAfterCancel" time="1.680" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
  <testcase name="fetchTheOrderDetailsNotCreated" time="0.206" classname="com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest"/>
</testsuite> <!-- com.nagarro.testCases.TestcasesForMoreThan3StopsInPresentFutureTest -->
